src/components/icons.tsx

```diff
@@ -0,0 +1,25 @@
+import { Spinner, SpinnerProps } from '@terra-ui-packages/components';
+import _ from 'lodash/fp';
+import React, { CSSProperties, ReactNode } from 'react';
+
+export { icon } from '@terra-ui-packages/components';
+
+export const centeredSpinner = (props: SpinnerProps = {}): ReactNode => {
```

Comment:
This really should be capitalized, but I didn't want to change all the files that call this.

Example usage:

<img width="365" alt="image" src="https://github.com/DataBiosphere/terra-ui/assets/484484/1f9e473d-2529-4b43-8a95-d49c2f5c4794">

Reply:
I would suggest keeping this as-is, since it's used as a render function and not as a component.